src/storage/copier/struct_column_chunk.cpp

```diff
@@ -117,6 +117,10 @@ void StructColumnChunk::setValueToStructField(
         fieldChunk->setValueFromString<int16_t>(
             structFieldValue.c_str(), structFieldValue.length(), pos);
     } break;
+    case LogicalTypeID::INT8: {
+        fieldChunk->setValueFromString<int8_t>(
```

Comment:
modify a csv file under tinysnb so it contains an int8_t field. This code path is used to support copying a int8_t filed in struct datatype.